platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -146,9 +146,10 @@ protected boolean doLogin() throws LoginException {
         try {
             callbackHandler.handle(callbacks);
         } catch (IOException ioException) {
+            LOGGER.debug("Exception while handling login.", ioException);
             throw new LoginException(ioException.getMessage());
         } catch (UnsupportedCallbackException unsupportedCallbackException) {
-            boolean result;
+            LOGGER.debug("Exception while handling login.", unsupportedCallbackException);
```

Comment:
❓ Is this debug message as necessary since you're throwing a checked exception on the next line that will presumably be logged elsewhere?